### ISSUE TITLE
Complete Checkout/ Create order

### ISSUE THREAD

**Issue by chirilavasiletiberiu** [Shopify/mobile-buy-sdk-ios#887]
Hello again. 
The app let's customer to choose two payment methods: Credit Card and On Account. I tried to use checkoutCompleteFree to create the Order from Checkout, but gives me an error saying that my checkout is not free (okay, I understand).
How should I create an Order from a Checkout and the payment to be done manually from the admin side?

Thank you.

**Comment by dbart01** [comment]
Am I correct in understanding that you're essentially looking for something like?
```graphql
mutation {
  checkoutCompleteWithManualPayment(checkoutId: "id") {
    checkout {
    	id
    }
    userErrors {
      field
      message
    }
  }
}
```
Where "manual payment" represents a transaction performed in-person, for example, using cash at a retail location.

**Comment by chirilavasiletiberiu** [comment]
Yes, the case you described is perfect. There is a possibility to accomplish this issue?

**Comment by dbart01** [comment]
This use case is not currently supported but it's something we'll look into.

**Comment by lorenzOliveto** [comment]
We're trying to do the same without success. 😩

**Comment by rebeccajfriedman** [comment]
We have added this issue to our backlog, and we hope to address it in the new year.

**Comment by helmiakbar** [comment]
Hi any update for this?

**Comment by rebeccajfriedman** [comment]
Unfortunately, we have no update at this time. The issue is still being tracked in our backlog.

**Comment by Komal2905** [comment]
Is this issue solved ?

**Comment by osamie** [comment]
> Is this issue solved ?

@Komal2905 Not yet, but we will be sure to provide an update once it's resolved
